prisma/schema.prisma

```diff
@@ -12,18 +12,7 @@ datasource db {
   shadowDatabaseUrl = env("SHADOW_DB_URL")
 }
 
-// Represents the name of a human individual in both Japanese and Romanji characters.
```

Comment:
My original data model for names had an unnecessary level of indirection, and I simplified it to work better with the GraphQL

Reply:
💯  this makes more sense to me